index.js

```diff
@@ -3,6 +3,8 @@ const pprops = require('p-props');
 
 module.exports = (obj, context, debug, log) => {
   if (debug) {
+    // should use console.log by default:
+    log = log || console.log;
```

Comment:
I think we should leave this out as not having a default. Make it a "required" attribute.